rules/scripts/sival_signature_test.bash

```diff
@@ -4,7 +4,9 @@ set -euo pipefail
 
 FILES_TO_CHECK=(@FILES@)
 
+VERIFY_ARGS=(@VERIFY_ARGS@)
+
 for file in "${FILES_TO_CHECK[@]}"
 do
-  @OPENTITANTOOL@ image manifest verify ${file}
+  @OPENTITANTOOL@ image manifest verify ${file} ${VERIFY_ARGS}
```

Comment:
Technically, VERIFY_ARGS is a list and you should use a list expansion.  Also, options should normally be placed before positional parameters (the commandline parser in opentitantool doesn't really care, but its kindof a nice convention):

```
@OPENTITANTOOL@ image manifest verify ${VERIFY_ARGS[@]} ${file}
```